CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
    kubefed2 disable now deletes the FederatedTypeConfig rather than set
```

Comment:
Since this is unreleased it should probably be `kubefedctl`

Reply:
 I wouldn't worry too much about that, it's necessary to edit the commit log anyway.